src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -179,17 +179,17 @@ const AxisClipSliders: React.FC<AxisClipSlidersProps> = (props) => {
   const updateRegion = (axis: AxisName, minval: number, maxval: number): void => {
     pauseOnInput(axis);
 
-    const { changeViewerSetting, numSlices, region } = props;
+    const { changeViewerSetting, numSlices } = props;
     // get a value from 0-1
     const max = numSlices[axis];
     const start = minval / max;
     const end = maxval / max;
-    changeViewerSetting("region", { ...region, [axis]: [start, end] });
+    changeViewerSetting("region", { [axis]: [start, end] });
```

Comment:
After talking this through, I was more able to wrap my head around it and I like the ability to make more granular updates with partials in this way.